### DISCUSSION TITLE
Timestep

### DISCUSSION THREAD

**Discussion by sthinius87** [deepmodeling/deepmd-kit#233] [Q&A]
Hello,
I am little confused about the timestep. Your default unit for the time step is 1ps, which can be quite a lot for AIMD.
Does it mean I need to prepare *raw-files in units of 1ps? or is there an option to set the time-step of the simulation?

Thanks for help
Sascha

**Comment by njzjz** [comment]
The time step is dependent on your system. You can set to any value. The default value is 1 fs?
By the way, that doesn’t mean you need to use every time step for training.

**Comment by jameswind** [comment]
Hi Sascha,

There is no need to specify the time step in the raw data file. The raw
data does not necessarily come from an MD/AIMD simulation. Any
misunderstanding?
